homework7.py

```diff
@@ -0,0 +1,28 @@
+"""just a dockstring"""
+
+# Zadanie1
+# 5948
+# 4 korovy i 4 byka
+W = 0
+while W <= 5948:
+    W += 1
+print(W, "= SUCCESS!")
+
+# zadanie3
+a = 2, 3, 4, 5, 6, 7, 8
```

Comment:
you do not need it, just a list [6, 2, 3, 8], based on this list missing numbers should be counted

Reply:
What if the list will be different?